### PR TITLE
DFS prefix changes

### PR DESCRIPTION
**Pull request by anmolanmol1234**
Change prefix check from blob to .blob. and dfs to .dfs.

### CODE REVIEW

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java [hunk 1]
```diff
@@ -29,14 +29,14 @@
...
   public static final String HTTP_SCHEME = "http";
   public static final String HTTPS_SCHEME = "https";
 
   public static final String WASB_SCHEME = "wasb";
   public static final String WASB_SECURE_SCHEME = "wasbs";
-  public static final String WASB_DNS_PREFIX = "blob";
+  public static final String WASB_DNS_PREFIX = ".blob.";
 
   private FileSystemUriSchemes() {}
 }
```
Comment (sreeb-msft) [inline hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java]: nit: EOF space

File: hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java [hunk 2]
```diff
@@ -758,7 +760,14 @@ public void testMkdirOnExistingFileWithImplicitParentDir() throws Exception {
   public void testCPKOverBlob() throws Exception {
     Assume.assumeTrue(getFileSystem().getAbfsStore().getPrefixMode() == PrefixMode.BLOB);
```
Comment (sreeb-msft) [inline hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java]: Can we now make the getAbfsStore() and getFileSystem() calls directly before this, as we are anyway needing to reuse this down the line?

File: hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java [hunk 3]
```diff
@@ -769,7 +778,14 @@ public void testCPKOverBlob() throws Exception {
   public void testCPKOverBlobEmptyKey() throws Exception {
     Assume.assumeTrue(getFileSystem().getAbfsStore().getPrefixMode() == PrefixMode.BLOB);
```
Comment (sreeb-msft) [inline hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java]: Same as the last comment for this one.

Comment (anmolanmol1234) [reply hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java]: Done

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java (hunk 1 above)
Comment (anmolanmol1234) [reply hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemUriSchemes.java]: Added

File: hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java [hunk 4]
```diff
@@ -22,6 +22,7 @@
 import java.lang.reflect.Field;
 import java.util.List;
 
+import org.apache.hadoop.fs.azurebfs.services.PrefixMode;
```
Comment (sreeb-msft) [inline hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java:25]: Import seems to be in the wrong place.

**Review by sreeb-msft — APPROVED** [review]
LGTM!

File: hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java (hunk 4 above)
Comment (anmolanmol1234) [reply hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java:25]: Added by IDE

**Pull request merged**